### DISCUSSION TITLE
running concurrently different docker images for DAG using k8sPodOperator

### DISCUSSION THREAD

**Discussion by y0zg** [apache/airflow#32654] [General]
Hi Team, 
We are currently using a single Docker image tag per environment to create a DAG with multiple tasks, and we're using the KubernetesPodOperator. We would like to have the ability to distinguish between images for different versions of DAG during runtime. For instance, if we have a running DAG with Image #1 and we update the DAG to use a new Image #2, we want to ensure that the old DAG continues to run with Docker Image #1. How can we achieve this? Do you have any recommendations?

**Comment by potiuk** [comment]
Store image in Xcom as result of previous task and rerun all but the first task tha produced the image tag
